packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -856,32 +861,79 @@ function findLocalizePositions(
       },
     });
   } else {
-    const traverseFast = ((types as unknown) as {
-      traverseFast: (node: types.Node, enter: (node: types.Node) => void) => void;
-    }).traverseFast;
-
-    traverseFast(ast, node => {
-      if (
-        node.type === 'TaggedTemplateExpression' &&
-        types.isIdentifier(node.tag) &&
-        node.tag.name === localizeName
-      ) {
-        const messageParts = utils.unwrapMessagePartsFromTemplateLiteral(node.quasi.quasis);
-        positions.push({
-          // tslint:disable-next-line: no-non-null-assertion
-          start: node.start!,
-          // tslint:disable-next-line: no-non-null-assertion
-          end: node.end!,
-          messageParts,
-          expressions: node.quasi.expressions,
-        });
-      }
+    traverse(file.ast, {
+      TaggedTemplateExpression(path) {
+        if (types.isIdentifier(path.node.tag) && path.node.tag.name === localizeName) {
+          const [messageParts, expressions] = unwrapTemplateLiteral(path, utils);
+          positions.push({
+            // tslint:disable-next-line: no-non-null-assertion
+            start: path.node.start!,
+            // tslint:disable-next-line: no-non-null-assertion
+            end: path.node.end!,
+            messageParts,
+            expressions,
+          });
+        }
+      },
     });
   }
 
   return positions;
 }
 
+let localizeOld: boolean | undefined;
+function unwrapTemplateLiteral(
+  path: NodePath<types.TemplateLiteral>,
+  // tslint:disable-next-line: no-implicit-dependencies
+  utils: typeof import('@angular/localize/src/tools/src/source_file_utils'),
+): [TemplateStringsArray, types.Expression[]] {
+  if (localizeOld === undefined) {
+    const { version: localizeVersion } = require('@angular/localize/package.json');
+    localizeOld = semverLt(localizeVersion, '10.1.0-rc.0', { includePrerelease: true });
+  }
+
+  if (localizeOld) {
+    // tslint:disable-next-line: no-any
+    const messageParts = utils.unwrapMessagePartsFromTemplateLiteral(path.node.quasi.quasis as any);
+
+    return [(messageParts as unknown) as TemplateStringsArray, path.node.quasi.expressions];
+  } else {
+    const [messageParts] = utils.unwrapMessagePartsFromTemplateLiteral(
+      path.get('quasi').get('quasis'),
+    );
+    const [expressions] = utils.unwrapExpressionsFromTemplateLiteral(path.get('quasi'));
+
+    return [messageParts, expressions];
+  }
+}
+
+function unwrapLocalizeCall(
+  path: NodePath<types.CallExpression>,
+  // tslint:disable-next-line: no-implicit-dependencies
+  utils: typeof import('@angular/localize/src/tools/src/source_file_utils'),
```

Comment:
seems like we're doing this in a bunch of places. It's better if we make it a regular import at the top of the file, but I'll leave this to you.